Platform/CoffeelakeBoardPkg/Script/StitchIfwi.py

```diff
@@ -368,6 +368,42 @@ def sign_binary(infile, stitch_dir, cfg_var):
     hash = bytearray(open('output/input/pubkey'+'.hash','rb').read())
     print(''.join('%02X ' % b for b in hash))
 
+def update_tpm_type(tpm_type, tree):
```

Comment:
Since we are now using this new method, plese remove the code in Line 408 and around which is already updating xml for TPM Type.

Reply:
Yes. will clean up the code.